.github/workflows/benchmarks.yml

```
@@ -21,12 +21,20 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Configure CMake
-      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DMR_MATH_ENABLE_BENCHMARK=ON -DMR_MATH_PRESET_BENCHMARK=ON
+    - name: Install Conan
+      uses: turtlebrowser/get-conan@main
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)